### DISCUSSION TITLE
Add `exif` extension

### DISCUSSION THREAD

**Discussion by atlza** [NativePHP/laravel#337] [Ideas]
Hi, thanks a lot for NativePhp,

I guess it would be nice to add Exif extension to the build as it is supported on all major platforms : https://static-php.dev/en/guide/extensions.html

Example code that fails : 
```
$fp = fopen($this->file , 'rb');
$headers = exif_read_data($fp);
```
with error : 
```
Call to undefined function exif_read_data()
```

Thanks again.

**Comment by simonhamp** [comment]
Thanks for the suggestion. I'm keen to keep the default PHP builds as streamlined as possible - every extension adds weight to the binaries and ultimately bloats every app, which means larger files, slower downloads, slower execution and a bigger security footprint.

There will be an officially-supported way to customise PHP builds for your app, it is coming it's just not at the top of the list yet.

**Reply by atlza** [reply]
Thanks for your answer. 
I tough it could be a good idea but I understand your point and I can manage to do without it.

And I prefer to see the Tauri versio on top of your list ;) 

This can be closed.
